src/v2/view-builder/views/phone/ChallengeAuthenticatorDataPhoneView.js

```diff
@@ -39,9 +39,17 @@ const Body = BaseForm.extend(
       const extraCssClasses =
         this.model.get('phoneNumber') !== loc('oie.phone.alternate.title', 'login') ?
           'strong no-translate' : '';
+      let nicknameText = this.model.get('nickname') || '';
+      if (nicknameText.length > 20) {
+        nicknameText = nicknameText.substring(0, 20) + '...';
+      }
+      if (nicknameText !== '') {
+        nicknameText = ' (' + nicknameText + ')';
```

Comment:
Should add a space before the `(` here, otherwise the phonenumber and nickname will run together.

Reply:
Addressed. Thanks